.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
+source = pyronear
```

Comment:
this needs to be changed to the actual package folder name

Reply:
Done